src/routing/FormTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default function FormTitle({
+  title,
+  subTitle,
+}: {
+  title: string;
+  subTitle?: string | undefined;
+}) {
+  return (
```

Comment:
I am going to use this in the introduction page as well so ill just grab yours now, along with the tests